packages/orchestration/src/exos/zcf-for-flow.js

```diff
@@ -26,6 +27,7 @@ export const ZcfForFlowsI = M.interface(
  * @param {VowTools} vowTools
  */
 export const prepareZcfForFlows = (zcf, zone, vowTools) => {
+  /** @satisfies {HostInterface<ZcfForFlows>} */
```

Comment:
👌 

some of the errors now are with the Host interface exposing methods that return promises. Should we excise those?

```
getTerms().issuers[0].isMyIssuer(): Promise<boolean>
makeEmptySeatKit(...).zcfSeat.getSubscriber(...): Promise<Subscriber<Allocation>>
```

I think otherwise they'll need durable watchers to become vows.

Reply:
Ugh yeah these will not be usable inside the membrane

Reply:
I'm inclined to drop `getTerms()` -- I don't see any need to delay calling it until a flow has started.

Reply:
excising `getSubscriber` means wrapping zcfSeat. I suppose that's reasonably straightforward.

Reply:
Wait I'm confused, zcfSeat is a local or remote object?

Reply:
zcfSeat is local, but the subscriber is from the zoe vat

Reply:
I'm trying to wrap ZcfSeat and running into all sorts of static typing issues around `Brand`s...
presumably due to `isMyIssuer()` returning a `Promise<boolean>`.
suggestions?

Reply:
These aren't just static typing issues, right? We have to remove that method through the attenuation or do some gymnastics to make it a vow.

Reply:
The type error does point to some runtime impact.

I just don't know how to remove methods from brands while preserving their identity, which is kinda the whole point of a brand

Reply:
I see. I think we can just remove them from the types so that they don't appear to be available. If someone goes and uses a method that the docs and static types say isn't there, they shouldn't be surprised that it fails. (This doesn't apply to when we need to attenuate powers, but that's not the case here. If the method worked we'd gladly include it.)

Reply:
> I think we can just remove them from the types so that they don't appear to be available.

That was my original thought but the problem is that passing such an object back as argument where a zcf seat is expected would cause a type error because the object wouldn't be compatible

Reply:
The attenuated ZCF should just be different from the ZCF. It just doesn't get past as an argument

Reply:
@dtribble the problem is not ZCF but `zcfSeat` in the result of `makeEmptySeatKit`. Does that need to be passed back anywhere? Should some of its methods be usable from the flow?